src/routes/search/components/SearchModal.js

```diff
@@ -65,13 +65,20 @@ class SearchModal extends React.Component<PropsType, StateType> {
     return categoriesWithTitle
       .filter(category => !category.isRoot())
       .concat(categoriesWithContent)
-      .map(category => ({ model: category, subCategories: [] }))
+      .map(category => ({
```

Comment:
Minor: Try to avoid code which passes the linting as the conses is that the style is alright :)

Reply:
No need to revert this though.